### ISSUE TITLE
protractorWrapper is not a function error

### ISSUE THREAD

**Issue by rayrapetyan** [Codeception/CodeceptJS#166]
In codeceptjs/lib/helper/Protractor.js:
`protractorWrapper = requireg('protractor').wrapDriver;`
"requireg" returns "undefined", because underneath:
`require.resolve('protractor')`
returns:
node_modules/protractor/built/browser.js
(and this module in turn exports wrapDriver as "Browser.wrapDriver").

So,
`protractorWrapper = requireg('protractor').Browser.wrapDriver;`
actually works, but fails somewhere later in tests.

Not sure how to fix that.

**Comment by rayrapetyan** [comment]
Created a relevant bug in protractor: https://github.com/angular/protractor/issues/3395

**Comment by rayrapetyan** [comment]
So it's a breaking change in Protractor 4.0, should be fixed in Codeception.

**Comment by cnishina** [comment]
So it looks like https://github.com/Codeception/CodeceptJS/blob/243eb5c4aa4f211ae95624151311da95f5a28568/lib/helper/Protractor.js#L84 should be changed to

``
   protractorWrapper = requireg('protractor').Browser.wrapDriver;

    EC = requireg('protractor').ExpectedConditions;
```

these won't work and should be changed since these are tied to the browser instance
```
    global.by = requireg('protractor').By;
    global.element = requireg('protractor').element;
```

after creating the browser, you can then access by and element....
https://github.com/Codeception/CodeceptJS/blob/243eb5c4aa4f211ae95624151311da95f5a28568/lib/helper/Protractor.js#L177
```
this.browser = protractorWrapper(this.browser, this.options.url, this.options.rootElement);
global.by = this.browser.by;
global.element = this.browser.element;
```

**Comment by DavertMik** [comment]
So what's the current state of it. Did Protractor revert their change in API or additional action is still required?

**Comment by DavertMik** [comment]
Ok, confirmed. Will be fixed on 0.4.0

**Comment by DavertMik** [comment]
Should be fixed in #171 
Protractor should be updated now
